### ISSUE TITLE
Does not escape slashes in heregex

### ISSUE THREAD

**Issue by netei** [eventualbuddha/decaffeinate#102]
Input: 

```
foo=///foobar/barfoo///
```
Output: 

```
var foo=/foobar/barfoo/;
```

expected:

```
var foo=/foobar\/barfoo/;
```

**Comment by eventualbuddha** [comment]
Thanks for continuing to find all these! Do you want to take a crack at
fixing this one? It should be fairly straightforward.

**Comment by eventualbuddha** [comment]
In particular, it's similar to how strings with interpolations get turned
into back ticks and have to escape back ticks within the contents.

**Comment by eventualbuddha** [comment]
Sorry, couldn't resist ;) Fixed in v1.41.5.

**Comment by netei** [comment]
If I find something new, I will give it a try :-)
